frontend/src/styles/_utils.scss

```diff
@@ -11,12 +11,23 @@
   font-weight: normal;
 }
 
+@mixin small-font {
+  font-size: .8rem;
```

Comment:
Was just wondering if there's any specific reason you chose these values for the sizes?

The small font looks great on my laptop screen but "feels" small on my desktop. I'm not familiar at all with the best practices for font size etc across devices, so maybe this isn't an issue.

<details>
<summary>Screenshots</summary>

## Before
![image](https://user-images.githubusercontent.com/45852430/230721806-87846304-a453-477e-a56d-fe9b34a93442.png)
## After
![image](https://user-images.githubusercontent.com/45852430/230721812-c222c922-b9a1-4849-a558-472c99e4adac.png)
</details>

Reply:
Thanks for testing on your desktop @vvidday!

The font sizes are defined using `rem` units (and monospace with `px` units), which are relative units and will look similar on every device. This is the same method we were using before this PR as well, but I have just replaced individual font size styling with mixin imports. 

The general practice is to use relative units like these instead of `cm`, `pt`, etc. However, there is still the issue of text appearing too small on desktops. The way to get around this is by using media queries, that set different font sizes based on the size of the device. 

Since the previous behavior of RepoSense was similar to this (not ideal on desktop), I think we can keep it as is for now and add media queries for responsiveness in a separate PR.

Reply:
Sounds good, thanks for clarifying!

Reply:
If you are specifically referring to the size of the commit messages, I have changed it to use the same font sizes as the code in the Code Panel, which is why it is smaller now.